src/tools/commands.ts

```diff
@@ -9,11 +9,9 @@ import { AISettingsModel } from '../models/settings-model';
  */
 export function createDiscoverCommandsTool(commands: CommandRegistry): ITool {
   return tool({
-    name: 'discover_commands',
```

Comment:
There is a `title` attribute if we want to keep something equivalent.
No sure there is any use case to it, though.

Reply:
Added in https://github.com/jupyterlite/ai/pull/245/commits/246dcf8ed84994f6df7999a291dd896e712ef87e